src/config/rcxml.c

```diff
@@ -855,6 +855,8 @@ entry(xmlNode *node, char *nodename, char *content)
 		rc.corner_radius = atoi(content);
 	} else if (!strcasecmp(nodename, "keepBorder.theme")) {
 		set_bool(content, &rc.ssd_keep_border);
+	} else if (!strcasecmp(nodename, "dropShadows.theme")) {
+		set_bool(content, &rc.shadows_enabled);
```

Comment:
Misses setting the default value in `rcxml_init()`, important when removing / commenting out / having a spelling mistake on an option and then doing a `Reconfigure`.